### ISSUE TITLE
Evaluate tsumegos with a DCNN

### ISSUE THREAD

**Issue by d180cf** [d180cf/tsumego.js#18]
Here is an interesting [paper](http://www.cs.cityu.edu.hk/~hwchun/research/PDF/Julian%20WONG%20-%20CCCT%202004%20a.pdf) that explains how to construct a simple NN to guess the correct move in 97% of cases. If mixed with a shallow 4-5 moves deep DFS, it could yield even better results. The plan is:

- [x] Use the sample tsumegos to generate a huge list of solved positions
- [x] Come up with an NN that would determine if the target block can be captured
- [x] Use TensorFlow to find the NN params
- [ ] use the NN params in the JS solver to evaluate positions

There are a few possible NN layouts:

- The naive linear model: `y = W x + b | softmax`
- A simple letter recognition model described in [the TF tutorial](https://www.tensorflow.org/get_started/mnist/pros)
- The net described in the paper above
- A net that also uses some precomputed features, e.g. whether a stone can be captured in a ladder, as described in [the AlphaGo paper](https://gogameguru.com/i/2016/03/deepmind-mastering-go.pdf)
